internal/pkg/scaffold/helm/chart_test.go

```diff
@@ -80,13 +79,9 @@ func TestCreateChart(t *testing.T) {
 		repoURLCharts = fmt.Sprintf("http://%s/charts/", testRepo.Addr)
 	)
 
-	var wg sync.WaitGroup
-	wg.Add(1)
+	errChan := make(chan error)
```

Comment:
Nit: improve the channel name since it is declared and used several hundred lines apart.
 ```suggestion
	testRepoServerErrChan := make(chan error)
```